### PR TITLE
Add Awesome-Nim logo to the list header

### PR DESCRIPTION
**Pull request by indiscipline**
Logo based on:

1. The [official Nim logo](https://github.com/nim-lang/assets/blob/master/Art/logo-design.svg) (the crown) by Joseph Wecker, used with permission from the Nim project.
2. [Awesome list logo](https://github.com/sindresorhus/awesome/blob/main/media/logo.svg) by Sindre Sorhus.
3. Font: [Orbitron](https://github.com/theleagueof/orbitron) by Matt McInerney.

Readme changes:
- Logo links to the Nim webpage per "Requirements for your Awesome list" (`The image should link to the project website or any relevant website`)
- Awesome badge aligned to the right per `Should be placed on the right side of the readme heading.`

### CODE REVIEW

**Comment by indiscipline** [comment]
Currently we fail the linter because of the divergence between the contribution rules and the codified linter rules.
See: https://github.com/sindresorhus/awesome/issues/2097

**Review by Mayjkhnskw88 — APPROVED** [review]

**Review by Lawrencesenin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -539,3 +537,6 @@ using `jester`.
 - [disruptek twitch](https://www.twitch.tv/disruptek) - The live broadcast regarding Nim language.
 - [yardanico twitch](https://www.twitch.tv/yardanico) - The live broadcast regarding Nim language.
 - [zachary_carter twitch](https://www.twitch.tv/zachary_carter) - The live broadcast regarding Nim language.
+
+---
+*Awesome-Nim logo is based on the "Nim Crown" logo by Joseph Wecker, used with permission from the Nim project.*
```
Comment (xflywind) [inline README.md:542]:
```suggestion
## License
*Awesome-Nim logo is based on the "Nim Crown" logo by Joseph Wecker, used with permission from the Nim project.*
```

File: README.md [hunk 2]
```diff
@@ -539,3 +537,6 @@ using `jester`.
 - [disruptek twitch](https://www.twitch.tv/disruptek) - The live broadcast regarding Nim language.
 - [yardanico twitch](https://www.twitch.tv/yardanico) - The live broadcast regarding Nim language.
 - [zachary_carter twitch](https://www.twitch.tv/zachary_carter) - The live broadcast regarding Nim language.
+
+## License
+*Awesome-Nim logo is based on the "Nim Crown" logo by Joseph Wecker, used with permission from the Nim project.*
```
Comment (xflywind) [inline README.md:542]:
```suggestion
Awesome-Nim logo is based on the "Nim Crown" logo by Joseph Wecker, used with permission from the Nim project.
```

File: README.md [hunk 3]
```diff
@@ -539,3 +537,6 @@ using `jester`.
 - [disruptek twitch](https://www.twitch.tv/disruptek) - The live broadcast regarding Nim language.
 - [yardanico twitch](https://www.twitch.tv/yardanico) - The live broadcast regarding Nim language.
 - [zachary_carter twitch](https://www.twitch.tv/zachary_carter) - The live broadcast regarding Nim language.
+
+## License
+Awesome-Nim logo is based on the "Nim Crown" logo by Joseph Wecker, used with permission from the Nim project.
```
Comment (xflywind) [inline README.md:542]:
```suggestion
---
*Awesome-Nim logo is based on the "Nim Crown" logo by Joseph Wecker, used with permission from the Nim project.*
```

**Comment by indiscipline** [comment]
Thanks for the merge. That issue mentioned above is rather awkward: completely ignored and swarmed by bot comments.

**Review by danieldll099 — COMMENTED** [review]
Toi

File: README.md (hunk 1 above)
Comment (abdullahixabad8) [reply README.md:542]: Wow I like it

**Review by JDKIATRA — APPROVED** [review]

**Review by devbyzq — COMMENTED** [review]
À finir pour mardi prochain

**Comment by 166dawe** [comment]
@indiscipline

**Pull request merged**